### ISSUE TITLE
Disabling modules should be done in a background job

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#5713]
### Is your feature request related to a problem? Please describe.

If you disable a module having many entries in the database, disabling the module with the browser can create a timeout which could corrupt the database.

### Describe the solution you'd like

Execute the module deactivation in a background job.

**Comment by luke-** [comment]
https://github.com/humhub/humhub/issues/5713

**Comment by funkycram** [comment]
@gevorgmansuryan is there a PR related to these commits?
`#5713` is not yet merged in https://github.com/humhub/humhub/blob/develop/CHANGELOG-DEV.md

**Comment by luke-** [comment]
@funkycram FYI, I've just merged the PR into `develop`

**Comment by funkycram** [comment]
Works perfectly! Thanks!

**Comment by funkycram** [comment]
@luke- @gevorgmansuryan I think we need a new method in `humhub\components\Module` to allow disabling things immediately (and not with cron), such as `$this->disableTheme();`.

See https://community.humhub.com/comment/perma?id=45889

**Comment by funkycram** [comment]
Thanks @yurabakhtin . Yes, it's a much better approach than adding an extra method!

**Comment by luke-** [comment]
> @luke- @funkycram @gevorgmansuryan Fixed in PR #6476:
> 
> I think the line https://github.com/humhub/humhub/pull/6179/files#diff-6e6e0e5fd7c6c82bc558a3f673725b3c6286b2ff9afe548347820b1b2fe8c009R116 - `Yii::$app->moduleManager->disable($module);` was added there by mistake because it looks like previous line `$module->disable()` was splitted to:
> 
> 1. queue - `Yii::$app->moduleManager->getModule($this->moduleId)->disable()`
> 2. controller - `Yii::$app->moduleManager->disable($module)`
> 
> I don't find a reason to keep the 2nd point because it is also called inside the end of the 1st code.
> 
> I also decided to inform user with the message after deactivate a module:
> 

Probably the line was added because otherwise after deactivation, the module is still listed as activated before the background job is finished.

As an idea: In the "Admin -> Modules" list, could we show the module also as `disabled` when:

```
if (QueueHelper::isQueued(new DisableModuleJob(['moduleId' => $moduleId]))) {
```

**Comment by yurabakhtin** [comment]
> As an idea: In the "Admin -> Modules" list, could we show the module also as `disabled` when:

@luke- Done in commit https://github.com/humhub/humhub/pull/6476/commits/f88512027f8e76220c939d16d81fe103bd6cbb3a.
